### PR TITLE
Updating jQuery to 3.2.1

### PR DESCRIPTION
**Pull request by marbor3**
Updating jQuery to the latest.
Right now found only two breaking changes - `$(document).on('ready'` removal and `data()` not returning  requested whole data object when data attribute name is in kebab-case.

Full list of changes and a migration guide:
https://jquery.com/upgrade-guide/3.0/

### CODE REVIEW

File: source/assets/js/helpers/estaticoapp.js [hunk 1]
```diff
@@ -71,23 +71,42 @@ class EstaticoApp {
...
-					modules = $element.data('init').split(' ');
-
-				modules.forEach((moduleName) => {
-					if (estatico.modules[moduleName] && !$element.data(moduleName + '-instance') && estatico.modules[moduleName].initEvents.indexOf(event.type) !== -1) {
-						this.initModule(moduleName, $element);
-					}
-				});
-			});
-		});
+		// jQuery 3 does not support `ready` event in $(document).on() https://jquery.com/upgrade-guide/3.0/#breaking-change-on-quot-ready-quot-fn-removed
```
Comment (orioltf) [inline source/assets/js/helpers/estaticoapp.js:105]: We should then remove the `ready` event from the [`module`](https://github.com/unic/estatico/pull/45/files#diff-bd637ae4c8e9321feb7e1d08e3ca0ed2R34) file, shouldn't we?

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "description": "Static site generator for frontend developers",
   "dependencies": {
     "bows": "1.4.8",
-    "jquery": "2.1.3",
+    "jquery": "3.1.1",
```
Comment (orioltf) [inline package.json]: In the meantime jQuery went up to v3.2.0. We should fix that right before finally having green light

**Review by orioltf — CHANGES_REQUESTED** [review]
I'd have a look at these 2 little things. otherwise the upgrade seems good :)

File: source/assets/js/helpers/estaticoapp.js (hunk 1 above)
Comment (marbor3) [reply source/assets/js/helpers/estaticoapp.js:105]: Not really - I still would keep module initialisation on some kind of event - so sending ready info like it was before - just now the info is just plain Object `{type: 'ready'}` to fit into other events sent later.

**Comment by orioltf** [comment]
Hi @marbor3 the Unit test from Slideshow fails. Could you have a look at it, please?

**Comment by backflip** [comment]
@orioltf, same issue as in https://github.com/unic/estatico/pull/64#issuecomment-307172434
Rebasing fixed the issue for me: https://deploy-preview-45--estatico.netlify.com/demo/modules/slideshow/slideshow.html → `> jQuery.fn.jquery`

**Review by backflip — APPROVED** [review]

**Pull request merged**